### PR TITLE
chore: add notes for developers on Windows

### PR DESCRIPTION
**Pull request by KexyBiscuit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
-Automation
-----------
+Notes for developers on Windows
+-------------------------------
 
-Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
+Currently, this repository cannot be checked out on Windows, since there're
```
Comment (MingcongBai) [inline README.md]: Please use "there are," and avoid using abbreviated forms in formal documentation.

File: README.md [hunk 2]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
+backslashes in file names, specifically required by [systemd.mount](https://www.freedesktop.org/software/systemd/man/latest/systemd.mount.html#Description)
```
Comment (MingcongBai) [inline README.md]: backslashes (\\)

File: README.md [hunk 3]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
+for [string escaping](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#String%20Escaping%20for%20Inclusion%20in%20Unit%20Names),
+which is [not allowed by Windows](https://learn.microsoft.com/windows/win32/fileio/naming-a-file#naming-conventions).
```
Comment (MingcongBai) [inline README.md]: not a legal file name in Windows filesystems

File: README.md [hunk 4]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
+Notebly, using MinGW environments to checkout the repository will fail either.
```
Comment (MingcongBai) [inline README.md]: Notably, repository checkout will also fail in MinGW environments (because?)

File: README.md [hunk 5]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
+We recommend the developer using Windows Subsystem for Linux (WSL) to checkout
```
Comment (MingcongBai) [inline README.md]: We recommend using using Windows Subsystem for Linux (WSL) to work on this repository.

File: README.md [hunk 6]
```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
...
+and to work on this repository. ICYMI, AOSC OS on WSL is available [on Microsoft Store](https://www.microsoft.com/store/apps/9NMDF21NV65Z)!
```
Comment (MingcongBai) [inline README.md]: In case you missed it -  please avoid using acronyms, I had to look this up too.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KexyBiscuit) [reply README.md]: Resolved.

File: README.md (hunk 2 above)
Comment (KexyBiscuit) [reply README.md]: I guess you mean (`\`), resolved.

File: README.md (hunk 4 above)
Comment (KexyBiscuit) [reply README.md]: The reason is that MinGW still works in Win32 subsystem, but that's out of scope of this document. I'm working on verifying if Cygwin will checkout the repository.

File: README.md (hunk 3 above)
Comment (KexyBiscuit) [reply README.md]: The Style Guide asks to replace "legal" with "valid". Also, it is a valid file name in NTFS or ReFS (I think), but not a valid file name in Windows (neither NT nor Win32 I think).

File: README.md (hunk 5 above)
Comment (KexyBiscuit) [reply README.md]: Resolved.

File: README.md (hunk 6 above)
Comment (KexyBiscuit) [reply README.md]: Resolved.

File: README.md (hunk 3 above)
Comment (MingcongBai) [reply README.md]: Sure, so:

not a valid Windows file name

File: README.md (hunk 4 above)
Comment (MingcongBai) [reply README.md]: > The reason is that MinGW still works in Win32 subsystem, but that's out of scope of this document. I'm working on verifying if Cygwin will checkout the repository.

Nice, further clarification would be best included

File: README.md (hunk 3 above)
Comment (KexyBiscuit) [reply README.md]: Resolved.

File: README.md (hunk 4 above)
Comment (KexyBiscuit) [reply README.md]: Cygwin doesn't help either.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**